app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> <%= @restaurants.name %> </h1>
```

Comment:
🔨 @AlExcel best practice is for this variable to be in singular tense since you are only returning one restaurant